server/routes/probationPractitionerReferrals/findStartPresenter.test.ts

```diff
@@ -47,29 +46,18 @@ describe('FindStartPresenter', () => {
 
   describe('fileInformation', () => {
     it('returns the file type and size in KB, rounded to one decimal place', async () => {
-      const presenter = new FindStartPresenter(
-        referrals,
-        { xlsx: 'example.xlsx', pdf: 'example.pdf' },
-        { bytes: 11126 },
-        loggedInUser
-      )
+      const presenter = new FindStartPresenter(referrals, { xlsx: 'example.xlsx' }, { bytes: 11126 }, loggedInUser)
 
       expect(presenter.fileInformation).toEqual('XLSX, 10.9KB')
     })
   })
 
   describe('structuredInterventionsDownloadHrefs', () => {
     it('prepends a slash to the passed-in filepaths', () => {
-      const presenter = new FindStartPresenter(
-        referrals,
-        { xlsx: 'example.xlsx', pdf: 'example.pdf' },
-        { bytes: 0 },
-        loggedInUser
-      )
+      const presenter = new FindStartPresenter(referrals, { xlsx: 'example.xlsx' }, { bytes: 0 }, loggedInUser)
 
       expect(presenter.structuredInterventionsDownloadHrefs).toEqual({
         xlsx: '/example.xlsx',
-        pdf: '/example.pdf',
```

Comment:
It seems like removing the PDF is against an [acceptance criterion](https://trello.com/c/IxPw3Ovj):

> Given that I am interested in learning what Structured Interventions are available
> When I view the PDF version of the spreadsheet
> Then it matches the current spreadsheet version

Is that ok?

Reply:
Confirmed this is OK, has now been removed from the AC